babel_runner.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/venv python3
+"""Script for handling translations with Babel"""
+from __future__ import annotations
+
+import argparse
+import os
+import subprocess
+
+import tomllib
```

Comment:
```suggestion
import argparse
import os
import subprocess
import tomllib
```

Reply:
I added this blank line because of ruff linter: https://github.com/python/python-docs-theme/actions/runs/10208999238/job/28250436250

Reply:
Try setting python-version for Ruff -- it is likely defaulting to something before 3.11

Reply:
You are correct. Ruff uses python-version from pyproject.toml, which is currently set to ">= 3.8".

Should I set ruff's target-version permanently, patch it for the workflow environment, or is there another way of passing the Python version to ruff via CLI?

Reply:
https://docs.astral.sh/ruff/configuration/#the-config-cli-flag

``--config "target-version=py311"``

Reply:
Well, the problem is that ruff check is failing is not just on my end, but in lint.yml action run, so AFAIK I'm not able to set the python version unless is replace python-version or set target-version in pyproject.toml.

I've worked around this issue using tomli as alternative (used your suggestion in babel add TOML support pull request). For this I added requirements.txt to use as single point of installing tomli or not based on Python version.